scripts/build-test-swarm.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+if [ -z "$1" ] || [ "$1" -le "0" ]; then
+    echo "Usage: $0 <number_of_peers>"
+    exit -1
+fi
+PEERS_NUM=$($1-1 | bc)
```

Comment:
Explain this please. 

On my PC this does not work.

Reply:
My mistake, yeah. Just wanted to decrement value via bc, e.g `echo 1-1 | bc` gives 0. I'll replace with `$((..))`